src/SimpleEventStore/SimpleEventStore.CosmosDb.Tests/AzureCosmosDbEventStoreInitializing.cs

```diff
@@ -26,47 +25,20 @@ public async Task when_initializing_all_expected_resources_are_created()
             var collectionName = "AllExpectedResourcesAreCreated_" + Guid.NewGuid();
             var storageEngine = await InitialiseStorageEngine(collectionName, collectionThroughput: TestConstants.RequestUnits);
 
-            var database = (await client.ReadDatabaseAsync(databaseUri)).Resource;
-            var collection = (await client.ReadDocumentCollectionAsync(UriFactory.CreateDocumentCollectionUri(DatabaseName, collectionName))).Resource;
-
-            var storedProcedure = client.CreateStoredProcedureQuery(collection.SelfLink)
-                .Where(r => r.Id.StartsWith("appendToStream-"))
-                .AsEnumerable()
-                .OfType<StoredProcedure>()
-                .Single();
-
-            var offer = client.CreateOfferQuery()
-                .Where(r => r.ResourceLink == collection.SelfLink)
-                .AsEnumerable()
-                .OfType<OfferV2>()
-                .Single();
-
-            Assert.That(storedProcedure, Is.Not.Null);
-            Assert.That(offer.Content.OfferThroughput, Is.EqualTo(TestConstants.RequestUnits));
```

Comment:
Why have these assertions been removed?  The indexing is optimised for event sourcing, so I'd like to ensure we are initialising the event store collection correctly.

Reply:
Same as the other one, I can't see a way to view those in the new SDK:
![image](https://user-images.githubusercontent.com/30480171/72159177-b65e0900-33b3-11ea-8f69-83e5efbf3a33.png)

Reply:
I've not had a chance to test this, but it looks like the following method gives you what you need - https://docs.microsoft.com/en-gb/dotnet/api/microsoft.azure.cosmos.container.readcontainerasync?view=azure-dotnet

The ContainerProperties class it returns provides the indexing policy

Reply:
Hi guys, 

I think the code to read the container properties (ttl, partition key and indexing policy) is:

var container = await collection.ReadContainerAsync();

The assertions could be:

Assert.That(expectedStoredProcedure, Is.Not.Null);
Assert.That(offer, Is.EqualTo(TestConstants.RequestUnits));
Assert.That(container.Resource.DefaultTimeToLive, Is.Null);
Assert.That(container.Resource.PartitionKeyPath, Is.EqualTo("/streamId"));
Assert.That(container.Resource.IndexingPolicy.IncludedPaths.Count, Is.EqualTo(1));
Assert.That(container.Resource.IndexingPolicy.IncludedPaths[0].Path, Is.EqualTo("/*"));
Assert.That(container.Resource.IndexingPolicy.ExcludedPaths.Count, Is.EqualTo(3));
Assert.That(container.Resource.IndexingPolicy.ExcludedPaths[0].Path, Is.EqualTo("/body/*"));
Assert.That(container.Resource.IndexingPolicy.ExcludedPaths[1].Path, Is.EqualTo("/metadata/*"));

Reply:
Ah thanks so much both! Re-added those assertions

Reply:
And about the other test that has been removed, I think it could be refactored in v3 as:

        [Test]
        public async Task when_initializing_with_a_time_to_live_it_is_set()
        {
            const int ttl = 60;
            var collectionName = "TimeToLiveIsSet_" + Guid.NewGuid();
            
           // TODO: Initiliase the storage engine with the _ttl_ parameter

            var database = client.GetDatabase(DatabaseName);
            var collection = database.GetContainer(collectionName);
            var container = await collection.ReadContainerAsync();

            Assert.That(container.Resource.DefaultTimeToLive, Is.EqualTo(ttl));
        }

Reply:
Yeah I re-added that test with the previous commit too :)